source/5_security.rst

```diff
@@ -381,3 +412,8 @@ At runtime, the system firmware may use a standard way - `SPDM <https://www.dmtf
 
 A platform should have a way to report a list of manifest and collect the runtime firmware measurement. As such, we can know the detailed firmware component information on a given platform.
 
+To supply the supply chain, the platform vendor should follow
```

Comment:
“supply the supply chain” sounds strange. Maybe *provide*?

Reply:
yes, typo.